src/main.rs

```diff
@@ -144,27 +146,34 @@ fn main() {
 					let debounce_config = config.clone();
 					let debounce = Debounce::default();
 
-					app_view(password, timeout_que_id, que, config.clone())
-						.any()
-						.window_title(|| String::from("Vault"))
-						.window_menu(|| {
-							Menu::new("")
-								.entry(MenuItem::new("Menu item"))
-								.entry(MenuItem::new("Menu item with something on the\tright"))
-							// menus are currently commented out in the floem codebase
-						})
-						.on_resize(move |r| {
-							let fn_config = debounce_config.clone();
-							debounce.clone().add(move || {
-								fn_config.general.write().window_settings.window_size =
-									(r.x1, r.y1);
-								let _ = fn_config.save();
-							});
-						})
-						.on_event(EventListener::WindowClosed, move |_| {
-							let _ = close_config.save();
-							EventPropagation::Continue
-						})
+					app_view(
+						password,
+						timeout_que_id,
+						que,
+						tooltip_signals,
+						config.clone(),
+					)
+					.any()
+					.window_title(|| String::from("Vault"))
+					.window_menu(|| {
+						Menu::new("")
+							.entry(MenuItem::new("Menu item"))
+							.entry(MenuItem::new("Menu item with something on the\tright"))
+						// menus are currently commented out in the floem codebase
+					})
+					.on_resize(move |event| {
```

Comment:
I know the convention for `on_event()` is to `|event|` but in this case it passes in a `Rect` not and `Event`. If `r` doesn't work, maybe `rect` might be better?